content/git/concepts/reset/reset.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: 'Reset'
+Description: 'In Git, reset is used to set the HEAD pointer back to a previous commit and optionally undo staged changes and the working tree.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'Version Control'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+In Git, the `reset` command is used to change the state of the Git repository or undo commits.
+
+## Syntax
+
+```
+git reset <mode-option> <commit>
+```
+
+The mode refers to the options for the reset command. These include:
```

Comment:
```suggestion
The `<mode-options>` refer to how far `reset` will go in changing the following contexts:

- Where the `HEAD` is pointing towards (usually done with just `git reset`).
- Whether the staging area, or Index, reflects the commit the `HEAD` is now pointing towards.
- Whether the working tree is rolled back to reflect the changes reset in the `HEAD` and staging area.

More specifically, these options include:
```

Reply:
Above this, we should also add a lv. 3 for that covers the `<commit-reference-number>` as well as the reserved `HEAD` keyword.

```
### Commits and the `HEAD` Pointer

Reply:
Do you want me to define those terms and place them above the options, under the following heading?

> ```
> ### Commits and the `HEAD` Pointer
> ```